dynamic/binary.go

```diff
@@ -31,6 +31,14 @@ func (m *Message) Marshal() ([]byte, error) {
 	return b.buf, nil
 }
 
+func (m *Message) MarshalAppend(b []byte) ([]byte, error) {
```

Comment:
Please add go doc to this exported method.

Reply:
oh yeah sure. Sorry I'm used to my linter screaming at me for this but it didnt go off here for some reason